### DISCUSSION TITLE
Failed to read DCP message browser format.

### DISCUSSION THREAD

**Discussion by johnoeo** [opendcs/opendcs#273] [General]
Hi community.
In the DCP message browser panel, when I try to show the messages, when I configure the display format and change the display in "Decoded" mode and in the "hydstra" decoding format, it is showing me the message of "java. lang.NullPointerException",; Can you guide me on how to fix the message.
Regards
![Failed_read_DCP_message_browser](https://github.com/opendcs/opendcs/assets/47491667/38e2078a-ce94-441c-9ad3-a9b47c7515be)

**Comment by MikeNeilson** [comment]
There should be a file called "gui.log" where you started the launcher from, it should have more information. Unfortunately there are several reasons that null pointer can show up and they aren't all obvious. 

It's also helpful if the launcher can be started with the `-d3` command line parameter to provide more information.
